git_python_systemd.yml

```diff
@@ -16,11 +13,14 @@
       environment:
         GIT_TERMINAL_PROMPT: 0
 
-    - name: write service file
-      shell: 'echo "[Unit]\nDescription=My test service\nAfter=multi-user.target\n\n[Service]\nType=simple\nRestart=always\nExecStart=/usr/bin/python3 {{ git_dir }}}/{{ repo | basename }}/{{ script }}\n\n[Install]\nWantedBy=multi-user.target" > /etc/systemd/system/python_script.service'
-
+    - name: Create service file from template
+      template:
+        src: python_script.service.j2
+        dest: /etc/systemd/system/python_script.service
+    
     - name: Start python script
       systemd:
         name: python_script
         state: started
         enabled: yes
+...
```

Comment:
Just like the start of .yml files have the optional `---`, you can choose to end with the optional `...`, which is interpreted as "end of document". Has limited functional uses and consider it a purely personal style suggestion.